src/data/finset.lean

```diff
@@ -2390,6 +2390,14 @@ by simpa only [disjoint.comm] using disjoint_bind_left t f s
   card (s ∪ t) = card s + card t :=
 by rw [← card_union_add_card_inter, disjoint_iff_inter_eq_empty.1 h, card_empty, add_zero]
 
+lemma card_sub_card {s t : finset α} (hst : t ⊆ s) :
+  card s - card t = (s.filter (∉ t)).card :=
```

Comment:
What is the difference between this and `card_sdiff` below?

Reply:
fixed